securebanking-openbanking-uk-rs-backoffice-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/backoffice/payment/calculate/elements/v3_1_5/internationalpayments/CalculateResponseElements.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2020-2022 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.securebanking.openbanking.uk.rs.api.backoffice.payment.calculate.elements.v3_1_5.internationalpayments;
+
+import com.forgerock.securebanking.openbanking.uk.error.OBErrorResponseException;
+import com.forgerock.securebanking.openbanking.uk.rs.api.backoffice.swagger.SwaggerApiTags;
+import io.swagger.annotations.*;
+import org.springframework.web.bind.annotation.*;
+import uk.org.openbanking.datamodel.error.OBErrorResponse1;
+import uk.org.openbanking.datamodel.payment.OBWriteInternationalConsent5;
+import uk.org.openbanking.datamodel.payment.OBWriteInternationalConsentResponse6;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+@Api(tags = {"v3.1.5", SwaggerApiTags.BACKOFFICE})
+@RequestMapping({"/backoffice/v3.1.5/international-payment-consents"})
+public interface CalculateResponseElements {
+
+    @ApiOperation(value = "Calculate payment consent response elements", response = String.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Response elements successfully calculated", response = String.class),
+            @ApiResponse(code = 400, message = "Bad request", response = OBErrorResponse1.class),
+            @ApiResponse(code = 401, message = "Unauthorized"),
+            @ApiResponse(code = 403, message = "Validation failed", response = OBErrorResponse1.class),
+            @ApiResponse(code = 404, message = "Not found"),
+            @ApiResponse(code = 404, message = "Method Not Allowed"),
+            @ApiResponse(code = 406, message = "Not Acceptable"),
+            @ApiResponse(code = 429, message = "Too Many Requests"),
+            @ApiResponse(code = 400, message = "Internal Server Error", response = OBErrorResponse1.class)
+    })
+    @RequestMapping(value = "/calculate-elements",
+            produces = {"application/json; charset=utf-8"},
+            method = RequestMethod.POST)
+    OBWriteInternationalConsentResponse6 calculateElements(
+            @ApiParam(value = "Create an Account Access Consents", required = true)
```

Comment:
Needs to be ResponseEntity<T> `Extension of HttpEntity that adds a HttpStatus status code. Used in RestTemplate as well @Controller methods.`
- ResponseEntity<OBWriteInternationalConsentResponse6>